src/widgets/principles/ui/principle-card/principle-card.scss

```
@@ -72,29 +25,43 @@
   display: flex;
   flex-flow: column nowrap;
   place-content: center flex-start;
-  align-items: center;
+  align-items: flex-start;
 
-  width: 380px;
-  height: 240px;
-  padding: 36px 24px 24px;
+  width: 100%;
+  padding: 23px;
 
   color: $color-gray-600;
 
-  background-color: #fffaf0;
+  background-color: $color-yellow-50;
   border: 1px solid rgb(255 219 32 / 8%);
   border-radius: 12px;
 
-  & .card-description {
-    margin-top: 16px;
+  & .card-header {
```

can you please remove all unnecessary parent selectors

```suggestion
   .card-header {
```

---

@Quiddlee I agree that this component should be refactored in some way, but the whole issue was not about a full refactor of the component but a fast bugfix, due to the inability to see the description of the card on mobile devices.

I think we are losing scope of the problem here.

---

in my opinion if we touch something we should consider refactor this but let's stick with what @ansivgit will say

---

It's minor fixes and in this case I agree with Bogdan